### DISCUSSION TITLE
T-echo Bricked?? Help Please :)

### DISCUSSION THREAD

**Discussion by Wizwooky** [meshtastic/Discussions#87] [nRF52]
T-echo bricked? Please help. After six months no issues one of my T-echos wont connect, wont bootload, wont restart/reboot. The only thing it does is when i hit the top button a red light flashes twice quickly.  other than that it seems non responsive. When trying to bootload the USB connect and disconnect  sound on my computer gos off continually. like its connecting to usb, then disconnected - repeat. Any ideas? Thank you 🙂
Just remembered I flashed to newest firmware about a week before the "brick".

**Comment by mariotti** [comment]
Issues with the batteries?

**Comment by eabase** [comment]
I know the LoRa32 module doesn't have ESD protection on USB ports. 
I can only guess that it was left out also on the T-Echo... (Please confirm!)

Leaving out ESD protection on USB and I/O should be a crime as it is so cheap to add.

**Reply by mariotti** [reply]
Can you confirm you changed the batteries?

**Comment by JimTheCowboy** [comment]
There seems to be an issue with T-Echos (NRF5240).

Try the factory erase, that should fix it. 

https://meshtastic.org/docs/getting-started/flashing-firmware/nrf52/nrf52-erase/

this could be a variant of https://github.com/meshtastic/firmware/issues/5491

**Comment by JimTheCowboy** [comment]
> T-echo bricked? Please help. After six months no issues one of my T-echos wont connect, wont bootload, wont restart/reboot. The only thing it does is when i hit the top button a red light flashes twice quickly. other than that it seems non responsive. When trying to bootload the USB connect and disconnect sound on my computer gos off continually. like its connecting to usb, then disconnected - repeat. Any ideas? Thank you 🙂 Just remembered I flashed to newest firmware about a week before the "brick".

This isssue should be solved with firmware 2.5.20 beta and upwards. You will need to do an NRF factory erase first. 
https://github.com/meshtastic/firmware/releases/tag/v2.5.20.4c97351

@fifieldt
